vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml

```diff
@@ -136,12 +136,22 @@
 - name: setup path to arkcase config file(Windows)
   become: yes
   become_user: pentaho
-  shell: sed -i 's~C:\\Users\\marjan\.stefanoski\\foia---reports-dw-1\\config\\arkcase_config\.properties~{{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/config/arkcase_config\.properties~g' {{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/misc/VARIABLE_CONFIG.ktr
+  lineinfile:
+    path: "{{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/misc/VARIABLE_CONFIG.ktr"
+    state: present
+    backrefs: yes
+    regexp: '^(\s*)<name>C:\\Users\\marjan\.stefanoski\\foia---reports-dw-1\\config\\arkcase_config.properties</name>'
+    line: '\1<name>{{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/config/arkcase_config.properties</name>'
 
 - name: setup path to arkcase config file(Linux)
   become: yes
   become_user: pentaho
-  shell: sed -i 's~/home/marst/projects/ETL-projects/EEOC/foia---reports-dw-1/config/arkcase_config\.properties~{{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/config/arkcase_config\.properties~g' {{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/misc/VARIABLE_CONFIG.ktr
+  lineinfile:
+    path: "{{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/misc/VARIABLE_CONFIG.ktr"
+    state: present
```

Comment:
Do these new regexps still work when `EEEOC` is in the line?

Reply:
No Dave, they did not work work for anything else beside that :(. I refactor and provide commit for including more wide search, and since "marst" is pretty specific term, we should be ok. I did couple of tests locally, it went ok.

Reply:
One last question ... does `*` work as expected, or do you need `.*`?

Reply:
I think that i hit the spot with the last commit. But, needed rearrange because the task "replace developer local path with good local path" was interfering on the task "setup path to arkcase config file(Linux)"